.github/workflows/genesis-verify.yml

```diff
@@ -0,0 +1,59 @@
+name: genesis-verify
+
+on:
+  pull_request:
+    branches:
+      - master
+    paths:
+      - "misc/deployments/**/genesis.json"
+
+jobs:
+  verify:
+    strategy:
+      fail-fast: false
+      matrix:
+        testnet: ["test4.gno.land", "test5.gno.land"]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Get changed files
+        id: changed-files
+        uses: tj-actions/changed-files@v41
+        with:
+          files: "misc/deployments/${{ matrix.testnet }}/genesis.json"
+
+      - name: Set up Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: "1.22"
+
+      - name: Install dependencies
+        run: go mod download
```

Comment:
This step is not needed, build will do this anyways

Reply:
83be3b2f0445fbb398afbf6537133325c11094b0